src/r2mm/profiles/ProfilesClient.ts

```diff
@@ -0,0 +1,34 @@
+import Axios, { AxiosResponse } from 'axios';
+import R2Error from 'src/model/errors/R2Error';
+
+export default class ProfilesClient {
+    public static TSCreateAPI = 'https://thunderstore.io/api/experimental/legacyprofile/create/'
+    public static TSRetrieveAPI = (profileImportCode: string) => {return `https://thunderstore.io/api/experimental/legacyprofile/get/${profileImportCode}/`}
+    public static HerokuRetrieveAPI = (profileImportCode: string) => {return `https://r2modman-hastebin.herokuapp.com/raw/${profileImportCode}`}
+
+    public static async sendProfile(endpoint: string, payload: string): Promise<AxiosResponse | R2Error> {

```

Comment:
Make sure to remove `| R2Error` from this type once you change the error handling to use throws instead of returns.